string signal
int32 lvalue
---
bool success

